### DISCUSSION TITLE
Toast messages for all clients and others for specific clients only

### DISCUSSION THREAD

**Discussion by jurassicjp** [Blazored/Toast#242] [Q&A]
Hello
First of all many thanks for your great NuGet.
We have a Blazor server application .Net 7.0 where manly everything is shown on all connected clients (browsers).
So we also have the Toast service as a singleton as each toast should be shown on all clients.

Now my question, how can I show a specific message (Toast(  only to the calling client and not to all clients?
Thanks for your help.
Best reagrds
JP

**Comment by chrissainty** [comment]
There isn't a built in way to handle this scenario. All toasts go through the single toast service so if that registered as a singleton then all clients will see all messages. You would need a scoped service to achieve per client toasts but there current components can't handle 2 toast services.

**Reply by jurassicjp** [reply]
Thanks a lot for your answer.
So there is no way to have toasts for all clients and other toasts for only the calling client in the same application?
Do you have a proposition how to solve this in Balzor?
